testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterBuilder.kt

```diff
@@ -25,13 +27,57 @@ class ClusterBuilder(clusterInfo: ClusterInfo, val REST_API_VERSION_PATH: String
         init {
             configureTracing("E2eClusterTracing", null, null, emptyMap())
         }
+        private val lock = ReentrantLock()
     }
 
     private val logger = LoggerFactory.getLogger("ClusterBuilder - ${clusterInfo.id}")
+    private val vNodeCreatorName = "vnodecreatoruser"
```

Comment:
Minor:
```suggestion
    private const val vNodeCreatorName = "vnodecreatoruser"
```
and we can probably move it inside `companion object`.